cms.php

```diff
@@ -4,23 +4,34 @@
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,
     PDO::FETCH_ASSOC);
 
-$query= $db->prepare("SELECT `email`, `phone`, `about`,`deleted`
+$query= $db->prepare("SELECT `email`, `phone`, `about`
 FROM `cms` WHERE `deleted` = 0 ORDER BY `id` DESC LIMIT 1;");
 
-$query1= $db->prepare("SELECT `project_id`, `project_img`, 
-`project_title`,`project_text`, `project_link` FROM `projects` WHERE 
-`project_delete` = 0 ORDER BY `project_id` ASC;");
+$query2= $db->prepare("UPDATE `cms` SET `deleted`= 1;");
 
+$query3= $db->prepare("INSERT INTO `cms` (`email`, `phone`, `about`)
+VALUES (:email_out, :phone_out, :about_out);");
+
+if (!empty($_POST)) {
```

Comment:
I would put all the preparing inside the `if` statement too. No need to `prepare()` if you aren't going to `execute()`

Reply:
You are right, it would be good if the prepare statement was in the if statement but page brings up an error if you do.